look-and-feel.md

```diff
@@ -0,0 +1,65 @@
+# Look and Feel
+
+### Name and wordmark
+
+Our name uses title case with the following variations:
+
+> **HYPHA**  
+> **HYPHA** Worker Co-operative  
+> Hypha Worker Co-operative Inc. [our legal name]  
+
+We use the following wordmark in online spaces, services and social media where we represent the organization (e.g., [Twitter](https://twitter.com/hyphacoop) and [LinkedIn](https://ca.linkedin.com/company/hyphacoop)). 
+
+<img src="images/hypha-wordmark.png" alt="Hypha Wordmark in white on purple background" width="50%"/>
+
+### Typography
+
+Our **heading typeface** is [Work Sans](https://fonts.google.com/specimen/Work+Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
+
+Heading typeface 
+>
+
+Body typeface 
+>
+
+Body typeface (web)
+> `font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial, sans-serif`
+
+### Colours
+
+<img src="images/hypha-colours.png" alt="Hypha colours" width="90%"/>
+
+Registration Black   
+`#000000;`  
+`rgb(0, 0, 0);`  
+
+Grey   
+`#777777;`   
+`rgb(119, 119, 119);`
+
+White  
+`#FFFFFF;`  
+`rgb(255, 255, 255);`  
+  
+Vivid Violet  
+`#9900FC;`  
+`rgb(153, 0, 252);`  
+
+Lime (Web) (X11 Green)  
+`#00ff00;`  
+`rgb(0, 255, 0);`  
+
+
+### Preferred icons and emoji
+
+We tend to use the following emoji and iconography: 
+
+🍄 `:mushroom:`  
+⚑  
```

Comment:
```suggestion
⚑  `&#9873;`
```

Reply:
Maybe include the 🏴 emoji as well?

Reply:
Sure, if we want to be anarchists about it ;)